docs/reference/commands.md

```diff
@@ -865,7 +865,7 @@ lakectl auth users credentials create [flags]
 
 ```
   -h, --help        help for create
-      --id string   user identifier (default: current user)
+      --id string   Username (email for password-based users). Defaults to current user.
```

Comment:
There are 3 places in this PR that we started adding '.' at the end of the help line of the argument. we need to align.